### ISSUE TITLE
infrastructure/testdriver/click_iframe_crossorigin.sub.html fails on Chrome macOS

### ISSUE THREAD

**Issue by Hexcles** [web-platform-tests/wpt#26295]
cc @stephenmcgruer @jgraham this is a somewhat mysterious macOS-only failure.

https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=56544&view=logs&j=af75b4de-b26f-5d4d-58e8-b41258076d44&t=052839da-7227-5391-fe2d-4b66ec169a0c&l=465
```
 2:33.45 TEST_START: /infrastructure/testdriver/click_iframe_crossorigin.sub.html
 2:33.71 pid:4081 [4083:775:1026/214353.293116:INFO:CONSOLE(3603)] "Uncaught Error: assert_unreached: click failed Reached unreachable code", source: http://web-platform.test:8000/resources/testharness.js (3603)
 2:34.05 TEST_END: Test OK. Subtests passed 0/1. Unexpected 1
FAIL TestDriver click on a document in an iframe - Uncaught Error: assert_unreached: click failed Reached unreachable code
    at http://web-platform.test:8000/infrastructure/testdriver/click_iframe_crossorigin.sub.html:15:9

 2:34.07 pid:4081 [4083:35587:1026/214353.647443:WARNING:gpu_process_host.cc(1265)] The GPU process has crashed 1 time(s)
 2:34.25 pid:4081 [4083:775:1026/214353.830957:WARNING:pref_notifier_impl.cc(40)] Pref observer for media_router.cast_allow_all_ips found at shutdown.
```

"The GPU process has crashed 1 time(s)" seems suspicious, but could be a red herring.

I'm going to create a metadata for this to temporarily suppress the failure.

**Comment by Hexcles** [comment]
Well, it's actually flaky: https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=56551&view=logs&j=af75b4de-b26f-5d4d-58e8-b41258076d44&t=052839da-7227-5391-fe2d-4b66ec169a0c&l=468

**Comment by Hexcles** [comment]
And yes, that's with Stephen's https://crrev.com/c/2481347 already merged.

**Comment by stephenmcgruer** [comment]
Yeah, the tracking https://crbug.com/1137228 has also continued to receive reports even after my attempted fix :(.

I haven't had time to dig in yet however.

**Comment by foolip** [comment]
This test seems flaky on Linux too, I just saw a random failure in https://community-tc.services.mozilla.com/tasks/O_h-U_sMSfqHKqdpdgwgCQ/runs/0/logs/public/logs/live.log

**Comment by Ms2ger** [comment]
I also seem to have had this fail on Firefox/mac: https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=86495&view=logs&j=af75b4de-b26f-5d4d-58e8-b41258076d44&t=85cf53c1-eea1-523e-d29b-83721b071df5
